scripts/lib.sh

```diff
@@ -6,6 +6,52 @@ require() {
     command -v "$1" >/dev/null 2>&1
 }
 
+sudo_askpass() {
+  if [ -z "${SUDO_ASKPASS-x}" ]; then
+    sudo --askpass "$@"
+  else
+    sudo "$@"
+  fi
+}
+
+# After using homebrew to install docker, we need to do some magic to remove the need to interact with the GUI
+# See: https://github.com/docker/for-mac/issues/2359#issuecomment-607154849 for why we need to do things below
+init_docker() {
+  # Need to start docker if it was freshly installed (docker server is not running)
+  if ! command -v docker &>/dev/null && [ -d "/Applications/Docker.app" ]; then
+    echo "Making some changes to complete Docker initialization"
+    # allow the app to run without confirmation
+    xattr -d -r com.apple.quarantine /Applications/Docker.app
+
+    # preemptively do docker.app's setup to avoid any gui prompts
+    sudo_askpass /bin/mkdir -p /Library/PrivilegedHelperTools
+    sudo_askpass /bin/chmod 754 /Library/PrivilegedHelperTools
+    sudo_askpass /bin/cp /Applications/Docker.app/Contents/Library/LaunchServices/com.docker.vmnetd /Library/PrivilegedHelperTools/
+    sudo_askpass /bin/cp /Applications/Docker.app/Contents/Resources/com.docker.vmnetd.plist /Library/LaunchDaemons/
+    sudo_askpass /bin/chmod 544 /Library/PrivilegedHelperTools/com.docker.vmnetd
+    sudo_askpass /bin/chmod 644 /Library/LaunchDaemons/com.docker.vmnetd.plist
+    sudo_askpass /bin/launchctl load /Library/LaunchDaemons/com.docker.vmnetd.plist
```

Comment:
Does this belong here? It's very specific to 1) mac os and 2) dev envs

Reply:
The condition of the if clause prevents Linux hosts from hitting this code.

Would you prefer it to be in a different module?

My long-term vision is to move most of the `bootstraph.sh` code into Sentry so all set up code is in the same place and make it easier for regressions to be caught in Sentry. I'm happy to discuss it further since perhaps we can just dispatch a workflow in `bootstrap-sentry` and test it there.